examples/credential-registry/src/lib.rs

```diff
@@ -704,8 +710,8 @@ pub struct RevocationDataOther {
 }
 
```

Comment:
`/// Key index in the revocation keys map` needs the `index` removed.

Reply:
Indeed. I thought I found all.